Created Spring 2024 Sub-folder in the Quiz Folder for Software Engineering + Added Quizzes

Muhammad-Daniyal-Jawad1 opened the pull request:
Made a subfolder in the Quiz folder for Software Engineering and Added 2 Quizzes of Spring 2024.

saleha-muzammil commented:
please mention the branch name you want this to merge into, kindly remove cheat  sheets, class activities

Muhammad-Daniyal-Jawad1 commented:
This is to be merged with the main branch of the repo.

saleha-muzammil commented:
Hello, I just checked all your files are .docx files, it has clearly been mentioned to upload pdfs.

Muhammad-Daniyal-Jawad1 commented:
Apologies, I have done so and converted other files in the Software Engineering folder into pdf files.

Muhammad-Daniyal-Jawad1 commented:
Created Quiz and Notes Sub-Folder in Compiler Construction Folder and Added Notes + Spring 2024 Quizzes. This is to be merged with the main branch.

Muhammad-Daniyal-Jawad1 commented:
Sorted the Operations Research Folder and put files at their respective sub-folders.

saleha-muzammil commented:
> Created Quiz and Notes Sub-Folder in Compiler Construction Folder and Added Notes + Spring 2024 Quizzes. This is to be merged with the main branch.

Please remove the notes

Muhammad-Daniyal-Jawad1 commented:
> > Created Quiz and Notes Sub-Folder in Compiler Construction Folder and Added Notes + Spring 2024 Quizzes. This is to be merged with the main branch.
> 
> Please remove the notes

Removed the Notes from the Compiler Construction Folder.
